cli.js

```diff
@@ -93,7 +93,16 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
         alias: 'd',
         describe: 'Data to send with invocation',
         type: 'string',
-      });
+      })
+      .example(
+`  // invoke a function
+  bn invoke foo
+
+  // invoke using JSON file data
+  bn invoke foo --json ./path/to/myfile.json
```

Comment:
I still find this confusing...

`json` does not convey the fact that it's a file. at first reading, each of these options might signify "from file" or "from string".

Maybe `--json-file`?

Reply:
I'll change it.

Reply:
btw, here's my proposal for all the options:

```
--json '[ 1, 2, 3 ]'
--json-file ./test.json
--data 'bare string'
```
WDYT?